CMakeLists.txt

```diff
@@ -10,22 +10,30 @@ set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
-set(COVERAGE "Coverage" ON)
+option(COVERAGE "Coverage" ON)
+
 set(COMPILE_FLAGS -Wall -Werror -fsanitize=address -fsanitize=leak -pthread)
 set(LINK_FLAGS -fsanitize=address)
 
 #SET(GCC_WERR_COMPILE_FLAGS "-Wall -Werror")
 #SET(CMAKE_C_FLAGS  "${CMAKE_C_FLAGS} ${GCC_WERR_COMPILE_FLAGS} -fsanitize=address")
 #SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pthread")
 
+include_directories(${libwebqc_SOURCE_DIR} ${libwebqc_SOURCE_DIR}/include)
+
+find_package(PkgConfig)
+
 find_package(CURL REQUIRED)
 include_directories(${CURL_INCLUDE_DIR})
 
+pkg_check_modules(CJSON REQUIRED libcjson)
```

Comment:
auto locate dependencies

Reply:
DOesn't work on  my 18.04 Ubuntu, they only added in 20.something